lib/ash/resource/relationships/belongs_to.ex

```diff
@@ -101,6 +102,10 @@ defmodule Ash.Resource.Relationships.BelongsTo do
                     type: :any,
                     default: Application.compile_env(:ash, :default_belongs_to_type, :uuid),
                     doc: "The type of the generated created attribute. See `Ash.Type` for more."
+                  ],
+                  through: [
+                    type: :any,
```

Comment:
I'm thinking spark should support `{:array, :atom}` 🤔 right...?

Reply:
`{:list_of, :atom}`

Reply:
😮